apps/sms/services/js/bridge_client_mixin.js

```diff
@@ -0,0 +1,144 @@
+/*global bridge */
+/*jshint esnext: true */
+
```

Comment:
It's veryyy wip, but the idea is to have code that deals with bridge client, upgrades, appInstanceId and can be shared across different ServiceXClients